services/generic/fake/generic.go

```diff
@@ -0,0 +1,36 @@
+// Package fake implements a fake generic service.
+package fake
+
+import (
+	"context"
+
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/generic"
+)
+
+func init() {
+	resource.RegisterService(
+		generic.API,
+		resource.DefaultModelFamily.WithModel("fake"),
+		resource.Registration[resource.Resource, resource.NoNativeConfig]{Constructor: func(
+			ctx context.Context,
+			deps resource.Dependencies,
+			conf resource.Config,
+			logger logging.Logger,
+		) (resource.Resource, error) {
+			return newGeneric(conf.ResourceName(), logger), nil
+		}})
+}
+
+func newGeneric(name resource.Name, logger logging.Logger) resource.Resource {
+	return &Generic{Named: name.AsNamed(), logger: logger}
+}
+
+// Generic is a fake Generic service that always echos inputs back to the caller.
```

Comment:
does this fake generic even echo inputs? I don't see a DoCommand anywhere

Reply:
It does not; good catch. I copied this comment from the fake generic component, which also does not actually echo inputs back to the caller. Modified the comment for both. If you think it's important functionality, I can also make them actually echo inputs.

Reply:
ya, let's make them actually echo inputs - hopefully should be pretty quick!

Reply:
Can do.

Reply:
Added `DoCommand` to both fake generic services and components and added small tests for them.